server/py_files/models/SentenceClassifier.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+from py_files.AccuracyAnalysis import AccuracyAnalysis
+
+class SentenceClassifier(object):
```

Comment:
@astrebeck : noting for discussion : I was more thinking on these lines for the base class of SentenceClassifier/Classifier (where we can dump all the common functionality - like generating scores or whatnot - and keep the model specific stuff inside the respective subclasses)

<serves 2 purpose: forces all classifiers to have the same structure and consistent methods - and repeated code goes inside the super class : and we still keep different classes, files, folders for maintainability>